### DISCUSSION TITLE
How to turn off cache ?

### DISCUSSION THREAD

**Discussion by limkaflyer** [kubilus1/autoortho#99] [Q&A]
As the title says, my cache folder keeps getting bigger. Is there a way to turn off the cache completely ?

**Comment by ivotaHR** [comment]
There is no option to auto delete cache. Delete by yourself when you reach a big amount data on your hdd/sdd.

**Comment by TinkerNZ** [comment]
What in terms of AO is a 'big amaount' ? I've been using it for just a couple of days and my cache is already 8 Gig+
What is the impact on how the app works if I delete some or all the cache contents?

Some clarification much appreciated.
TinkerNZ

**Reply by jonaseberle** [reply]
You can delete any files from the cache directory. The app will only redownload if needed.

**Comment by dpdurst** [comment]
I see the same after about 7 days its close to 9GB, no biggie I have plenty of space. Maybe adding something in the code that checks cache or an option to say if cache is X big auto delete then refresh would be helpful. Something automated so you can set a size and not worry about doing it manually

**Comment by BinaryOutlook** [comment]
Just delete yourself weekly? Mines are near 100GB.

**Reply by dpdurst** [reply]
Is there a maximum it can go or just unlimited based on drive space? Yeah I could delete manually, but if something could be done to automate through the program why not? I guess I could write a script and run it once a month as well if needed. Just tossing about idea's

**Reply by BinaryOutlook** [reply]
Just go unlimited as far as I am aware of the current state of this program

**Comment by Simssmith** [comment]
One more question about cache. How smart is the system?  If it downloaded a tile in the past but a new 'version' of the tile is available,  will it download the new version?  Is the old one overwritten?  If I fly same route all the time, the cache would stop growing right?

**Reply by TinkerNZ** [reply]
Excellent question and adds on to mine above about what impact deleting part or all of the cache has. 
The whole question of cache management needs to be addressed something to provide users better access and control would be a big bonus.

**Reply by BinaryOutlook** [reply]
The cache will be accessed again if you fly the same route, hence no cache increase.

**Reply by jonaseberle** [reply]
Currently cache files look like this: `53909_38592_16_BI.jpg`. So regarding your question: Just like Ortho4XP there is no way to know if Bing data has been updated (without re-downloading it!). If the tile config changes (e.g. Bing -> Arc), the image will be redownloaded and saved under a new name. All the other Ortho4XP rules apply: So for example a mesh change does not require to redownload imagery.

@TinkerNZ the impact will only be that you freed some space. It is just a "transparent" cache, like a layer between you and the provider: If a file is found locally it is taken, if not it is downloaded.

**Comment by jonaseberle** [comment]
See also https://github.com/kubilus1/autoortho/discussions/81
